app/src/pages/inside/common/defectTypeTooltip/defectTypeTooltip.jsx

```diff
@@ -53,48 +53,49 @@ export class DefectTypeTooltip extends Component {
 
     const defectConfig = projectConfig.subTypes && projectConfig.subTypes[type.toUpperCase()];
 
+    const filteredBodyData = Object.keys(data)
```

Comment:
I think it would be better to move this sequence to the `getFilteredBodyData` function and call it here.

Reply:
Fixed.